### PR TITLE
add test for or_swap()

### LINKED ISSUE #260: test the `or_swap()` function
Create unit tests for `or_swap()` that it actually swaps the two reflections or raises exceptions when the number of reflections is not exactly two.

### PR DESCRIPTION
**Pull request by prjemian**
- close #260

### CODE REVIEW

File: hkl/user.py [hunk 1]
```diff
@@ -158,7 +158,19 @@ def new_sample(nm, a, b, c, alpha, beta, gamma):
 
 
 def or_swap():
```
Comment (mrakitin) [inline hkl/user.py:160]: Just a general comment about the function name - it's too short and can be confusing, i.e. is it "or" or "orientation"? Not sure in how many places it's used already, but maybe it's not too late to rename it to something like "swap_reflections"...

**Review by mrakitin — APPROVED** [review]
The test looks good to me. A bit unhappy about the function name, but it's a topic for a separate PR.

File: hkl/user.py (hunk 1 above)
Comment (prjemian) [reply hkl/user.py:160]: The name has been in use for decades, in another popular diffractometer software package.  It is recognized.

Comment (prjemian) [reply hkl/user.py:160]: In this context, `or` is for **o**rientation **r**eflection (it's not a logic operator here).  There are other `or` usages.

**Pull request merged**